library/pod-security-policy/flexvolume-drivers/template.yaml

```diff
@@ -16,10 +16,12 @@ spec:
           properties:
             allowedFlexVolumes:
               type: array
+              description: "An array of AllowedFlexVolume objects with the single field: 'driver'."
```

Comment:
No need to list the fields of objects, that's done by the schema.

Reply:
Shortened.